### PR TITLE
Release PR for 2.4.2 patch

### PR DESCRIPTION
**Pull request by jfy133**
- [#824](https://github.com/nf-core/eager/issues/824) Fixes large memory footprint of bedtools coverage calculation.
- [#822](https://github.com/nf-core/eager/issues/822) Fixed post-adapterremoval trimmed files not being lane-merged and included in downstream analyse

Linting WILL fail because I had to pin the nf-core tools version, you can check for other errors manually if you want

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`). [DONE MANUALLY]
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]

File: lib/NfcoreSchema.groovy [hunk 1]
```diff
@@ -259,7 +259,7 @@ class NfcoreSchema {
         return new_params
     }
 
-    /*
+     /*
```
Comment (drpatelh) [inline lib/NfcoreSchema.groovy:262]:
```suggestion
    /*
```

**Review by drpatelh — APPROVED** [review]
LGTM! Tiny change in comment required. Dunno where that came from.

**Comment by drpatelh** [comment]
Need to fix the linting too.

File: lib/NfcoreSchema.groovy (hunk 1 above)
Comment (jfy133) [reply lib/NfcoreSchema.groovy]: Nope, this results in the second linting failure...

image

**Comment by jfy133** [comment]
Linting is just that I hardcoded the last DSL1 version of tools in the CI otherwise it always fails with a bunch of other DSL2 related issues

**Review by TCLamnidis — APPROVED** [review]
LGTM

**Pull request merged**